accelerometer/summariseEpoch.py

```diff
@@ -405,8 +413,91 @@ def calculateECDF(e, inputCol, summary):
         summary[inputCol + '-ecdf-' + str(accUtils.formatNum(x,0)) + 'mg'] = \
             accUtils.formatNum(ecdf, 5)
 
+def calculatePSD(e, summary):
+    """Calculate the power spectral density from fourier analysis of a 1 day frequency
+    
+    :param pandas.DataFrame e: Pandas dataframe of epoch data
+    :param dict summary: Output dictionary containing all summary metrics
 
+    :return: Write dict <summary> keys 'PSD-<W/Hz>'
+    """
+    # collects the sleep column from data frame assumes sleep if it is the highest imputed value
+    y = (np.argmax(e[['sleepImputed', 'moderateImputed', 'sedentaryImputed',
```

Comment:
Can this be updated so that y='sleepImputed' if available (not assuming sleep is highest imputing value) ... or else, y= 'accImputed' instead (i.e. if 'sleepImputed' is not available)?